en/components/components-overview.md

```diff
@@ -1,30 +1,40 @@
+
+<img class="responsive-img" src="../images/components-page-figma.png" srcset="../images/components-page-figma@2x.png 2x" />
 
 ## Sketch
 
 The Sketch libraries have a `🧩 Components` page containing all Components and symbols related to them. When selecting a Symbol instance, through the Override section in the right panel, one can change the content inside e.g. enabled/disabled states, icons, colors, etc. Below is a small part of showing some of the Components and symbols such as the different types of [Buttons](button.md), [Button Group](button-group.md) and form elements like [Checkbox](checkbox.md), [Switch](switch.md), and [Radio Group](radio-group.md).
 
-<img class="responsive-img" src="../images/components-page.png" />
+<img class="responsive-img" src="../images/components-page-sketch.png" srcset="../images/components-page-sketch@2x.png 2x" />
 
 ## Adobe XD
 
 In the Adobe XD library you can find the `🧩 Components` in the middle column. Here, the various states of a component can be switched from the right panel. Also, we have used the `Stack` and `Padding` to align and distribute elements vertically or horizontally and change their position. When you want to swap a nested component with another one, you can simply drag it from the `Libraries` panel and drop it on top of the current one to replace it. You will also notice that some components, such as the Grid, for example, use a `Repeat Grid` to speed configuration up. Below you may find some of the Components like different types of [Buttons](button.md), [Button Group](button-group.md) and form elements such as [Checkbox](checkbox.md), [Switch](switch.md), and [Radio Group](radio-group.md).
 
-<img class="responsive-img" src="../images/components-page-xd.png" />
+<img class="responsive-img" src="../images/components-page-xd.png" srcset="../images/components-page-xd@2x.png 2x" />
 
 > [!WARNING]
-> Detaching a component from the library in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Detaching a component from the library in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers containing metadata descriptions of the state, template, and other properties of the component. In Figma libraries, there is one additional layer placed in every component named `🚫 componentVersion` containing the version, which helps for successful code generation.
```

Comment:
I think it's better to separate the info for Figma from the info for Sketch like it's done with Adobe XD.

Reply:
This is a warning box where is a summary for detaching. By default, it was for both Sketch and Adobe, and this is why I followed the same logic, but yes, I can split it somehow; just let me know which will be the optimal @sdimchevski